controls/anssi.yml

```diff
@@ -622,6 +622,7 @@ controls:
     status: automated
     rules:
     # CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
+    - kernel_config_arm64_default_mmap_min_addr
```

Comment:
An alternative solution would be: Don't create a new rule, instead, have a single rule which would be applicable on multiple architectures and wouldn't use a template and would have a more complex OVAL that would check for architecture and give different results based on architecture.

Reply:
@Mab879 Do you think that this would be better than the current solution in this PR?

Reply:
I think the complex OVAL might make sense. So the other arches don't get lost.

Reply:
But, the current solution uses a template so the current solution benefits from the shared code.

How will the other arches get lost?

Reply:
By authors forgetting about others arches when creating profiles.